autoprotocol/container_type.py

```diff
@@ -1170,6 +1170,42 @@ def row_count(self):
     cat_no="4513",
 )
 
+#:
+ROUND384LVNC = ContainerType(
+    name="384-well round-bottom low-volume Corning-4512 plate",
+    well_count=384,
+    well_depth_mm=6.58,
+    well_volume_ul=Unit(20.0, "microliter"),
```

Comment:
I see `35ml` from the DB for the `ROUND384LV` !!

Reply:
For `ROUND384LV` in the code above, it says `20ml`; make sense why I had `20 ml` since I had just copied it from above. I'll change both!